Misc/Copy-ItemFromBcContainer.ps1

```diff
@@ -0,0 +1,66 @@
+﻿<# 
+ .Synopsis
+  Copy an item (file or folder) from a container to the host system.
+ .Description
+  Copy an item (file or folder) from a container to the host system.
+ .Parameter containerName
+  Name of the container from which you want to copy the item.
+ .Parameter containerPath
+  Path to the item in the container which has to be copied.
+ .Parameter localPath
+  Path on the host where the item will be placed.
+  If the source item is a folder, 'localPath' will be created (or treated) as a folder and all items inside the source folder will be copied to it.
+ .Example
+  Copy-ItemFromBcContainer -containerName test2 -containerPath "c:\run\my" -localPath "c:\temp\copy-of-my"
+#>
+function Copy-ItemFromBcContainer {
+    Param (
+        [string] $containerName = $bcContainerHelperConfig.defaultContainerName,
+        [Parameter(Mandatory = $true)]
+        [string] $containerPath,
+        [Parameter(Mandatory = $false)]
+        [string] $localPath = $containerPath
```

Comment:
localPath should be mandatory true and not defaulted

Reply:
While I do agree, I mimicked what is done in `Copy-FileFromBcContainer`. But maybe there it's purely for backward compatibility? Anyways ... will fix.